browser/react/components/Yoonah.js

```diff
@@ -15,9 +16,14 @@ export default class Yoonah extends React.Component {
   render () {
     return (
       <a-entity id="room" position="0 0 0">
```

Comment:
Not the first place this occurs probably but just noticed, do we need this `id`?